libretroshare/src/util/rsdebug.cc

```diff
@@ -208,3 +208,75 @@ void rslog(const RsLog::logLvl lvl, RsLog::logInfo *info, const std::string &msg
 		lineCount++;
 	}
 }
+
+
+//From https://codereview.stackexchange.com/a/165162
+/**
+ * @brief hex_dump: Send Hexadecimal Dump to stream
```

Comment:
No need to duplicate documentation in the .cc file keep it only in the header

Reply:
Also it seems you positioned this code down between the deprecated functions put it before the line thet says something like "all code after this line is deprecated"

Reply:
Done.